From 0b8600414976a5ad733604c7b1428071baf239c2 Mon Sep 17 00:00:00 2001
From: BuffaloWill <wvandevanter@gmail.com>
Date: Wed, 22 Apr 2020 17:00:14 -0400
Subject: [PATCH] Updated route

Big thanks to Antonious Papdopoulos (https://github.com/uyatashi) for pointing out this issue.
---
 routes/admin.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/routes/admin.rb b/routes/admin.rb
index 9b7d6f2..0ee5cbc 100644
--- a/routes/admin.rb
+++ b/routes/admin.rb
@@ -56,6 +56,7 @@
 
 # create backup of all attachments
 get '/admin/attacments_backup' do
+  redirect to('/no_access') unless is_administrator?
   bdate = Time.now
   zip_file = './tmp/Attachments' + '-' + (bdate.strftime('%Y%m%d%H%M%S') + '.zip')
   Zip::File.open(zip_file, Zip::File::CREATE) do |zipfile|
